btc.com/src/endpoint/balance.ts

```diff
@@ -1,26 +1,31 @@
 import { balance } from '@chainlink/ea-factories'
-import { Requester } from '@chainlink/external-adapter'
-import { Config } from '@chainlink/types'
 import { isChainType, isCoinType } from '.'
+import * as blocktrail from 'blocktrail-sdk'
```

Comment:
> Blocktrail is now btc.com?
> 
> If this is true could we document it somehow, as it's confusing reading it as is.

I'm not sure how this is all settling on their side.
Blocktrail's website now links to BTC.com and BTC.com's [new docs](https://dev.btc.com/docs/js) say to use the SDK.
Their support email shows their name as bitmain.



> What would be the benefit of accessing the API through this SDK?
Originally I did it because of https://dev.btc.com/docs/js#api-authentication (they hadn't documented how to authenticate so I just went through the SDK), but I _could_ go back now and try sending the API_KEY and API_SECRET as a param.